app/konnect/gateway-manager/troubleshoot.md

```diff
@@ -3,6 +3,41 @@ title: Troubleshoot Data Plane Nodes
 content_type: how-to
 ---
 
+
+## Upgrading managed data planes
+
+**Problem** 
+
+The data plane version is out of date. 
+
+**Solution**
+
+With Dedicated Cloud Gateways, {{site.base_gateway}} updates can be performed with the {{site.konnect_short_name}} manager or API. {{site.konnect_short_name}} will automatically provision new data plane nodes and slowly migrate traffic while monitoring for errors. For more information, read the [Upgrade documentation](/konnect/gateway-manager/data-plane-nodes/upgrade/).
+
+## DNS capabilities
+
+**Problem**
+
+APIs need to be availble publicly for external applications and privately for internal traffic. 
+
+**Solution**
+
+Cloud Gateways offer both a **Public** and **Private** mode. In public mode, you can expose your APIs to the internet, 
```

Comment:
```suggestion
Cloud Gateways offer both a **Public** and **Private** mode.
```
Looks like the second half of this sentence is missing, and then the description below covers both - so I think it's better to remove.